#! /bin/sh /usr/share/dpatch/dpatch-run
## 21manpage_nethack_activation.dpatch by  <hesso@pool.math.tu-berlin.de>
##
## DP: Document that there are two possible causes for 'nethack on'
## DP: being activated by default...

@DPATCH@
--- screen-4.0.3.orig/doc/screen.1	2007-08-06 23:39:11.000000000 +0200
+++ screen-4.0.3/doc/screen.1	2007-08-06 23:40:24.000000000 +0200
@@ -2260,7 +2260,8 @@
 .I screen
 was compiled with the NETHACK flag defined. The
 default setting is then determined by the presence of the environment 
-variable $NETHACKOPTIONS.
+variable $NETHACKOPTIONS and the file ~/.nethackrc - if either one is present,
+the default is \fBon\fP.
 .sp
 .ne 3
 .B next
--- screen-4.0.3.orig/doc/screen.texinfo	2007-08-06 23:39:12.000000000 +0200
+++ screen-4.0.3/doc/screen.texinfo	2007-08-06 23:43:15.000000000 +0200
@@ -4444,7 +4444,8 @@
 This option is only available if @code{screen} was compiled with the
 NETHACK flag defined (@pxref{Installation}). The default setting is then
 determined by the presence of the environment variable
-@code{$NETHACKOPTIONS}.
+@code{$NETHACKOPTIONS} and the file @code{~/.nethackrc} - if either one is
+present, the default is @code{on}.
 @end deffn
 
 @node Nonblock, Number, Nethack, Miscellaneous
